### PR TITLE
Cardy updates ii

### PR DESCRIPTION
**Pull request by davidpmccormick**
__Adding__
- FreeSticker
- Map
- FeaturedText
- TabNav

### CODE REVIEW

File: common/views/components/FeaturedText/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Purpose
+
+Used to emphasise a paragraph of text. The first para on an exhibition page, for example.
```
Comment (jamesgorrie) [inline common/views/components/FeaturedText/README.md]: Should examples maybe be a section? Bikeshedding for a bigger cardy convo though.

Comment (davidpmccormick) [reply common/views/components/FeaturedText/README.md]: Maybe, yeah. I've been trying to leave a link to edit the readmes on GitHub to make it easier for non-devs to edit. Obviously forgot to do so here. Will update.

**Comment by davidpmccormick** [comment]
> is it now part of it when we create a component, no matter how incomplete or likely to change it is, it goes in cardy?

I feel like we had a fairly good flow with this back when there was a lot of new and different-from-each-other UI components being built (the good ol' Fractal days). I think you're right – we need to remember to be in the habit of getting stuff into Cardy asap (and we've got a WIP section, so there's not a good reason not to do it that way).

**Comment by jamesgorrie** [comment]
Yea, so maybe it is a start in cardy approach.

**Pull request merged**